### PR TITLE
[W33] Luke: 08/10 - 08/14

### PR DESCRIPTION
**Pull request by luke-chang**

### CODE REVIEW

File: Luke.md [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
-* [MultiScreen]
-    - helped test the boundry issue on TV
-        - https://github.com/luke-chang/gaia/tree/multiscreen_private
+        - encountered an unexpected problem and waiting for feedback
 
 ### This Week ###
 
+* [TV remote control]
+    - Proof of concept
+        - write a demo page to test the performance between Ajax and WebSocket
```
Comment (evelynhung) [inline Luke.md]: do you have a link of your demo page?

File: Luke.md [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
+
+* [SUPL NI]
+    - [WIP] Bug 1195602 - [GPS] Support SUPL NI case in Gaia
```
Comment (evelynhung) [inline Luke.md]: Why do you still work on this? @@

File: Luke.md (hunk 1 above)
Comment (luke-chang) [inline Luke.md]: Yeah. https://github.com/luke-chang/ajax-websocket-test
And I'll attach it in the next weekly report as well

File: Luke.md (hunk 2 above)
Comment (luke-chang) [inline Luke.md]: It's about time to land it on v2.2 as well as unit tests since gecko part has gotten r+ last week.

**Pull request merged**